README.md

```diff
@@ -529,6 +530,64 @@ console.log(JSON.stringify(work))
  */
 ```
 
+### Handle flow deletion
+
+Deleting a flow cannot be done by adding the flow in the `destructiveChanges.xml` and deploy.
+A [known issue](https://issues.salesforce.com/issue/a028c00000gAwixAAC/deletion-of-flow-metadata-through-destructive-changes-not-supported) exist to cover this feature.
+Please do not assume committing a flow metadata deletion to the repo, and then run sgd will allow you to delete a flow.
+
+We suggest to deal with flow deletion in one go by following those steps (it requires the `FlowDefinition` metadata which is not avalable in API `v44+`)
+1. Set the `FlowDefinition` `activeVersionNumber` to `0`
+2. List the `FlowDefinition` in the `package.xml`
+3. List all the existing version of the `Flow` in the `destructiveChanges.xml` (can be fetch via SOQL using this query : `SELECT FlowDefinitionView.ApiName, VersionNumber, Status FROM FlowVersionView WHERE FlowDefinitionView.ApiName='<FLOW_API_NAME>'`)
+4. Deploy this `FlowDefinition` with a `package.xml` and post delete all the `Flow` versions with a post `destructiveChangesPost.xml` 
```

Comment:
**suggestion:** do you think you could deal with this automation inside `sfdx-hardis` @nvuillam ?

I think it could be wrap behind a command like `sf hardis org delete flow -n <FLOW_API_NAME>`, what do you think @nvuillam ?